app/controllers/reviewables_controller.rb

```diff
@@ -74,6 +74,7 @@ def index
           total_rows_reviewables: total_rows,
           types: meta_types,
           reviewable_types: Reviewable.types,
+          unknown_reviewable_types: Reviewable.pending.distinct.pluck(:type) - Reviewable.sti_names,
```

Comment:
I think we need to display these in a less technical way if possible, since we are showing admins the class names for these reviewable types directly. I can't recall if you told us, but is there any way to show which plugin they are from?

If not, we should convert the type a bit to something resembling English, so taking your screenshot example of `ReviewableAiChatMessage`, we should display it as `Ai chat message`. Not perfect but at least a bit easier on the eyes than the direct class. Potentially even say if a segment of the text is < 2 chars then automatically upcase for `AI chat message`.

If you can get the plugin name, then the format `PluginName - Type` would be nice.

Reply:
Also you could add a method to `Reviewable` for the types themselves, e.g.:

```ruby
def self.unknown_types
  Reviewable.pending.distinct.pluck(:type) - Reviewable.sti_names
end
```

Then do the formatting to plainer language elsewhere